### DISCUSSION TITLE
How to Hangup the call  after completing the conversation or by saying word like "bye"?

### DISCUSSION THREAD

**Discussion by orion-satyammishra** [voxos-ai/bolna#328] [Q&A]
I have added in prompt:
          "call_cancellation_prompt":"You are an helpful AI assistant that's having conversation with customer on a phone call. Based on the given transcript, should you cut the call?\n\n  RULES: 1. If user is not interested in talking, or is annoyed or is angry we might need to cut the phone. 2. You are also provided with original prompt use the content of original prompt to make your decision. For example if the purpose of the phone call is done and we have all the required content we need to cut the call." 

 "hangup_after_LLMCall": true,

Please provide info about if I need to add anything else.

**Comment by h3110Fr13nd** [comment]
Yes, I couldn't make it work as well.
